xsnap/makefiles/mac/xsnap-worker.mk

```diff
@@ -151,9 +179,9 @@ $(OBJECTS): $(PLT_DIR)/xsPlatform.h
 $(OBJECTS): $(SRC_DIR)/xsCommon.h
 $(OBJECTS): $(SRC_DIR)/xsAll.h
 $(OBJECTS): $(SRC_DIR)/xsScript.h
+$(OBJECTS): $(XS_TLS_DIR)/fdlibm/math_private.h
 $(OBJECTS): $(SRC_DIR)/xsSnapshot.h
 $(OBJECTS): $(INC_DIR)/xs.h
-$(OBJECTS): $(EXTRA_DEPS)
```

Comment:
This is an agoric-sdk build mechanism to force rebuild on config changes, please do not remove.